fun-police/systemPatterns.org

```diff
@@ -94,25 +94,24 @@ class PaymentRepository(Protocol):
    - Contains implementation logic for data persistence, service integration, etc.
    - may be non-deterministic, but must be idempotent
 
-2. *Temporal Activity Implementation* (=sample/repos/temporal/__init__.py=):
+2. *Temporal Activity Implementation* (=sample/repos/activities.py=):
    - Created using =@temporal_activity_registration= decorator.
    - Automatically wraps all async methods with =@activity.defn=
    - Imported and instantiated in worker for registration
 
 3. *Workflow Proxy* (=sample/repos/temporal/proxies/payment.py=):
+   - Created using =@temporal_workflow_proxy= decorator
+   - Automatically generates methods that call =workflow.execute_activity()=
    - Used inside workflows for determinism
-   - Calls =workflow.execute_activity()=
    - No actual business logic
    - Ensures workflow replay safety
 
 *Decorator Implementation Details*:
 
-The =@temporal_activity_registration= decorator creates explicit Temporal Activity classes.
-The recommended approach is to define these classes in the temporal package's
-=__init__.py= file for clarity:
+The =@temporal_activity_registration= decorator creates explicit Temporal Activity classes:
```

Comment:
I want to talk more about this - but will do so in the next PR (where I'll move just these classes in the sample code to match what I've done in julee_example, which *I* think is clearer - let's see what you think.